### DISCUSSION TITLE
Markdown to PDF: Convert using Half Letter paper size

### DISCUSSION THREAD

**Discussion by johnmapeson** [jgm/pandoc#10113] [Q&A] [answered]
Is it possible to convert from Markdown to PDF using the Half Letter (5.5 by 8.5 inches) paper size?

I tried

```
pandoc -V papersize:halfletter -o output.pdf input.txt
```

But this gives an error: 

> [makePDF] LaTeX Warning: Unused global option(s): [halfletterpaper]

**Comment by badumont** [comment]
LaTeX supports only b5paper out of the box among the B formats. Fortunately, the "geometry" variable allows you to use the LaTeX package of the same name, which considerably extends the range of supported predefined paper sizes:

  pandoc -V geometry:b6paper -o output.pdf input.md

**Reply by johnmapeson** [reply]
Hello, thanks a lot. This is my fault, I completely re-wrote the question after it was posted. (To my excuse, I did it very quickly.)

The current version of the question is about how to produce a PDF with the Half Letter paper size, which is 5.5 by 8.5 inches. Neither `pandoc -V papersize:halfletter -o output.pdf input.txt` nor `pandoc -V geometry:halfletterpaper -o output.pdf input.md` works for me.

**Comment by badumont** [comment, marked as answer]
Half letterpaper is not a predefined size in geometry, but you can provide arbitrary dimensions, e.g.:

  pandoc -o output.pdf -V geometry:paperwidth=2cm -V geometry:paperheight=12cm input.md

By the way, the documentation of almost all LaTeX packages can be found here: https://ctan.org/

It may be more convenient to set the "geometry" variable in a metadata header as shown in the Pandoc manual.

**Reply by johnmapeson** [reply]
Thanks!
